### PR TITLE
SISRP-12147 - SIR card is beneath Tasks and Notifications in mobile view - expected?

### PR DESCRIPTION
**Pull request by christianvuerings**
https://jira.berkeley.edu/browse/SISRP-12147

*Note* GL3 change

### CODE REVIEW

**Comment by christianvuerings** [comment]
QA PR at #4624

File: src/assets/templates/dashboard.html [hunk 1]
```diff
@@ -1,15 +1,15 @@
-<div class="medium-5 columns medium-push-3 cc-page-dashboard-column-b">
+<div class="medium-3 columns ">
```
Comment (johncrossman) [inline src/assets/templates/dashboard.html]: Trailing space after 'columns'. A future PR tweak perhaps.

**Comment by johncrossman** [comment]
Onward!

File: src/assets/templates/dashboard.html (hunk 1 above)
Comment (christianvuerings) [inline src/assets/templates/dashboard.html]: @johncrossman follow up at #4629

**Pull request merged**